src/Activity.js

```diff
@@ -1,48 +1,59 @@
-    let selectedDayData = userRepo.chooseDayDataForAllUsers(this.activityData, date);
-    return parseFloat((selectedDayData.reduce((acc, elem) => acc += elem[relevantData], 0) / selectedDayData.length).toFixed(1));
-  }
-  userDataForToday(id, date, userRepo, relevantData) {
-    let userData = userRepo.getDataFromUserID(id, this.activityData);
-    return userData.find(data => data.date === date)[relevantData];
-  }
-  userDataForWeek(id, date, userRepo, releventData) {
-    return userRepo.getWeekFromDate(date, id, this.activityData).map((data) => `${data.date}: ${data[releventData]}`);
+// userRepo is used as parameter for individual user or whole userRepo
+// dataType string which determines which type of date to accesse
+
+
+class Activity { //should probably be renamed
+  constructor(activityData) { //constructor sucks but has to stay
+    this.activityData = activityData //
+  }
+  //// begin uneeded methods
+  // getMilesFromStepsByDate(id, date, userRepo) {
+  //   // THIS IS NEVER USED
+  //    //returns miles walked on a specific date
+  //   let userStepsByDate = this.activityData.find(data => id === data.userID && date === data.date); //user data that corresponds to user and input date
+  //   return parseFloat(((userStepsByDate.numSteps * userRepo.strideLength) / 5280).toFixed(1)); //how is userRepo.strideLength even being accessed?
+  //   // userRepo is array of user objects but how does that have a strideLength property?
+  //   // userRepo is improperly named
+  // }
+  // // not used
+  // getActiveMinutesByDate(id, date) { //return total minues active on a date for a specific user
+  //   let userActivityByDate = this.activityData.find(data => id === data.userID && date === data.date);
+  //   return userActivityByDate.minutesActive;
+  // }
+  // calculateActiveAverageForWeek(id, date, userRepo) { //return average active minutes for all users
+  //   return parseFloat((userRepo.getWeekFromDate(date, id, this.activityData).reduce((acc, elem) => { //acc and elem are terrible names
+  //     return acc += elem.minutesActive;
+  //   }, 0) / 7).toFixed(1));
+  // }
+  // accomplishStepGoal(id, date, userRepo) { // returns true if user steps = daily step goal (should be >=)
+  //   let userStepsByDate = this.activityData.find(data => id === data.userID && date === data.date);
+  //   if (userStepsByDate.numSteps === userRepo.dailyStepGoal) { //>= not ===
+  //     return true;
+  //   }
+  //   return false
+  // }
+  // getDaysGoalExceeded(id, userRepo) { // returns array of dates user exceeding goal steps
+  //   return this.activityData.filter(data => id === data.userID && data.numSteps > userRepo.dailyStepGoal).map(data => data.date);
+  // }
+  // getStairRecord(id) { // returns highest number of stairs user as ever done on single day
+  //   return this.activityData.filter(data => id === data.userID).reduce((acc, elem) => (elem.flightsOfStairs > acc) ? elem.flightsOfStairs : acc, 0);
+  // }
+  //// end not needed methods
```

Comment:
Some of these might need to be used eventually when we get our activity section more user friendly